Tests/PeripheralTest+CharacteristicOperation.swift

```diff
@@ -62,7 +62,7 @@ class PeripheralCharacteristicOperationsTest: BasePeripheralTest {
     }
     
     func testWriteValueWithResponse() {
-        let characteristic = _Characteristic(characteristic: createCharacteristic(uuid: "0x0001", service: service), peripheral: peripheral)
+        let characteristic = _Characteristic(characteristic: createCharacteristic(uuid: "0x0001", service: service), peripheral: peripheral)!
```

Comment:
Have you considered adopting [XCTUnwrap](https://developer.apple.com/documentation/xctest/3380195-xctunwrap) and then making the function throwable?
```suggestion
        func testWriteValueWithResponse() throws {
            let characteristic = try XCTUnwrap(_Characteristic(characteristic: createCharacteristic(uuid: "0x0001", service: service), peripheral: peripheral))
```

Reply:
I have made the change you can have a look.